src/CoreMedia/CMSync.cs

```diff
@@ -374,19 +378,21 @@ bool IsDeprecated ()
 		{
 #if IOS || MONOMAC
 			return PlatformHelper.CheckSystemVersion (GetMethodDeprecatedMajor, GetMethodDeprecatedMinor);
-#elif TVOS
+#elif TVOS || WATCH
 			return true;
 #endif
 		}
 
 		public CMTimebase CopyMasterTimebase ()
 		{
-			IntPtr ptr;
+			IntPtr ptr = IntPtr.Zero;
 			bool deprecated = IsDeprecated ();
 			if (deprecated)
 				ptr = CMTimebaseCopyMasterTimebase (Handle);
+#if !WATCH
 			else
 				ptr = CMTimebaseGetMasterTimebase (Handle);
+#endif
```

Comment:
only the deprecated version was made available on watchOS 6 ?

Reply:
I'm stupid, I wanted to do the opposite 🤦‍♂

Reply:
@spouliot The code is confusing: `IsDeprecated` doesn't mean "Use the deprecated API", but the opposite (something like "Are we running on an OS version where the old API has been deprecated; i.e. we can use the new API").

Reply:
@rolfbjarne ok, so I did the right thing \o/

Reply:
k, make sense - in a twisted way 🔀